syntax = "proto3";

package targetshooting;

import "google/protobuf/empty.proto";

service Shooting {
  rpc DimeCentroDiana (google.protobuf.Empty) returns (CentroReply) {}

  rpc DisparaCannon (DisparaRequest) returns (DisparaReply) {}  
}

message CentroReply {
  int32 distance = 1;
}

message DisparaRequest {
  string username = 1;
  int32 angle = 2;
  int32 speed = 3;
}

message DisparaReply {
  float shoot_distance = 1;
}
